ceph.spec.in

```diff
@@ -544,6 +544,7 @@ Requires:       python%{python3_pkgversion}-PyJWT
 Requires:       python%{python3_pkgversion}-Routes
 Requires:       python%{python3_pkgversion}-Werkzeug
 %endif
+Suggests:     python%{python3_pkgversion}-saml
```

Comment:
it's available in fedora and openSUSE Tubleweed though. but with different names. see https://pkgs.org/search/?q=python3-saml

i am not sure what "python2dist(python3-saml)" implies. if it means "python3-saml", it's okay. otherwise could update accordingly?

Reply:
Thanks for pointing that out, @tchaikov! Will fix it.